.github/workflows/license-check.yml

```diff
@@ -0,0 +1,27 @@
+name: license-check
+
+on:
+  pull_request:
+    branches: [v2/*]
```

Comment:
Why not `[v?/*]`? or `[v*/*]`?

The idea is that we should check license for every version in the future also. Otherwise we'll keep changing this file...